config/vufind/facets.ini

```diff
@@ -304,3 +304,31 @@ visual_facets = "callnumber-first,topic_facet"
 ; Use facet field names as keys and untranslated facet values as values.
 [ShowFacetValue]
 ;format[] = "Book"
+
+; This section can be used to define custom filters, where a simple value
+; in the VuFind URL gets remapped to a more complex filter in the Solr
+; request, or where special Solr filters are applied UNLESS a simple value
+; is added to the VuFind URL
+;
+; This can be useful in combination with checkbox filters and hidden filters
+; if you wish to define filters that are too complex to easily represent in
+; .ini file syntax, and it also makes URLs more readable and easier to share.
+[CustomFilters]
+; This is the name of the virtual Solr field used when applying custom
+; filters. For example, if you define an inverted filter named include_books
+; below, and your custom filter field was the default of __custom__, you
+; would add &filter[]=__custom__:include_books to the URL in order to apply
+; the inverted filter.
+custom_filter_field = "__custom__"
```

Comment:
Would it be possible to use "vufind" as the default custom filter field? It would look nicer. Also, it would be nice to include an example of how to set up a checkbox filter to use the field, and in its configuration 'vufind' would look nicer and be easier to get right than double underscore.

Reply:
Agreed. I have made the change (and added the example... though the comment on the example is perhaps too short because I didn't want to go multi-line; open to improvements if you have any suggestions).

Reply:
...actually, the comment was bothering me, so I revised it. Hopefully better now!